homeassistant/components/duotecno/light.py

```diff
@@ -48,9 +48,11 @@ async def async_turn_on(self, **kwargs: Any) -> None:
         else:
             # restore state
             val = None
-        await self._unit.set_dimmer_state(val)
+        if not self.is_on:
```

Comment:
Is this tested?
With this change we will not send the update to the duotecno smartbox is there is just a brightness change,

lets say the device is on for 30%, and you want to change this to 50%, the pyDuotecno API is not called as is_on will return True.